WordPress/Classes/ViewRelated/Comments/CommentDetailInfoViewController.swift

```diff
@@ -100,24 +104,23 @@ extension CommentDetailInfoViewController: UITableViewDelegate {
     }
 }
 
-// MARK: - DrawerPresentable
-extension CommentDetailInfoViewController: DrawerPresentable {
-    var collapsedHeight: DrawerHeight {
-        .intrinsicHeight
-    }
-
-    var allowsUserTransition: Bool {
-        false
-    }
-
-    var compactWidth: DrawerWidth {
-        .maxWidth
-    }
-}
-
-// MARK: - ChildDrawerPositionable
-extension CommentDetailInfoViewController: ChildDrawerPositionable {
-    var preferredDrawerPosition: DrawerPosition {
-        .collapsed
+extension CommentDetailInfoViewController {
+    func show(from presentingViewController: UIViewController, sourceView: UIView) {
+        let navigationController = UINavigationController(rootViewController: self)
+        if UIDevice.isPad() {
```

Comment:
If you put the app on iPad into split screen, this branch will be executed and the view controller will be presented as a full screen modal instead of popover. Maybe we should check horizontal size class instead?

Reply:
Fixed. I changed it to `presentingViewController.traitCollection.horizontalSizeClass == .regular`.

Btw, the Split View is pretty much completely broken on iPad as you can't access any of your site menus.

<img width="400" alt="Screenshot 2024-08-01 at 12 11 24 PM" src="https://github.com/user-attachments/assets/6ca27a53-dddb-4060-a8c7-750eb7bacd5c">